addons/packages/packetfence.spec

```diff
@@ -1082,26 +1061,8 @@ fi
 %config                 /usr/local/pf/conf/caddy-services/pfsso.conf
 %config                 /usr/local/pf/conf/caddy-services/httpdispatcher.conf
 %dir                    /usr/local/pf/conf/monitoring
-%config(noreplace)      /usr/local/pf/conf/monitoring/carbon.conf
-                        /usr/local/pf/conf/monitoring/carbon.conf.example
-%config(noreplace)      /usr/local/pf/conf/monitoring/collectd.conf.rhel
-                        /usr/local/pf/conf/monitoring/collectd.conf.rhel.example
-%config(noreplace)      /usr/local/pf/conf/monitoring/collectd.conf.debian
-                        /usr/local/pf/conf/monitoring/collectd.conf.debian.example
-%config(noreplace)      /usr/local/pf/conf/monitoring/dashboard.conf
-                        /usr/local/pf/conf/monitoring/dashboard.conf.example
-%config(noreplace)      /usr/local/pf/conf/monitoring/local_settings.py.rhel
-                        /usr/local/pf/conf/monitoring/local_settings.py.rhel.example
-%config(noreplace)      /usr/local/pf/conf/monitoring/local_settings.py.debian
-                        /usr/local/pf/conf/monitoring/local_settings.py.debian.example
-%config(noreplace)      /usr/local/pf/conf/monitoring/statsd_config.js
-                        /usr/local/pf/conf/monitoring/statsd_config.js.example
-%config(noreplace)      /usr/local/pf/conf/monitoring/storage-schemas.conf
-                        /usr/local/pf/conf/monitoring/storage-schemas.conf.example
-%config(noreplace)      /usr/local/pf/conf/monitoring/storage-aggregation.conf
-                        /usr/local/pf/conf/monitoring/storage-aggregation.conf.example
-%config(noreplace)      /usr/local/pf/conf/monitoring/types.db
-                        /usr/local/pf/conf/monitoring/types.db.example
+%config(noreplace)      /usr/local/pf/conf/monitoring/netdata.conf
+                        /usr/local/pf/conf/monitoring/netdata.conf.example
```

Comment:
Shouldn't the plugins be part of the spec?

Reply:
Done